sootty/utils.py

```diff
@@ -10,5 +12,165 @@ def dec2anybase(input, base, width):
         else:
             res += chr(rem - 10 + ord("A"))
         input = int(input / base)
-
+ 
     return res[::-1].zfill(ceil(log(2**width - 1, base)))
+
+def vcdid_hash(s):
```

Comment:
For each of these functions, can you add a short comment at the top that explains what they're supposed to do.